test/levelup-test.js

```diff
@@ -17,11 +17,11 @@ describe('levelup', function() {
 
 		zstreams
 			.fromArray(testArray)
-			.pipe(db.createWriteStream())
+			.pipe(db.createWriteStream(), { objectMode: true })
```

Comment:
Ah pipeOpts only have one option on them. Yeah it should be documented that the options can the options for instantiating streams while wrapping.